.circleci/config.yml

```diff
@@ -222,12 +230,14 @@ jobs:
       - run:
           name: Report tests and coverage
           command: |
-            cargo llvm-cov --json --output-path workspace/test-results/cov.json test --features=emulator --features=bigtable --jobs=2
+            cargo llvm-cov --json --output-path workspace/test-results/cov.json nextest --features=emulator --features=bigtable --jobs=2 --profile=ci
       - store_artifacts:
           path: ~/project/workspace/test-results/cov.json
           destination: cov.json
```

Comment:
We need to have the xml report stored as an artifact as well in this step so that we can download it from the API. The `store_test_results` makes things observable in CircleCI, but we also want to scrape the xml atm.

I wonder if you'd be willing to make this change for the integration tests too as part of this PR? It's a bit of scope creep, but I noticed this is missing there too.

Reply:
I thought of this but wasn't sure. I can make the change to both, i think thats fine

Reply:
See https://github.com/mozilla-services/autopush-rs/pull/748/commits/60c6c0c14dc3a8f5ae650df65d2832994cb645e5